apps/worker/src/app/workflow/usecases/send-message/send-message.usecase.ts

```diff
@@ -159,9 +153,7 @@ export class SendMessage {
       identifier: command.job.identifier,
     });
 
-    const data = await this.matchMessage.getFilterData(messageMatcherCommand);
```

Comment:
Call MessageMatcher to get the filter data (subscriber and payload) to then send it again to the MessageMatcher doesn't make much sense to me.